tests/PullRequest.py

```diff
@@ -402,3 +407,70 @@ def testUpdateBranch(self):
             self.pull.update_branch("addaebea821105cf6600441f05ff2b413ab21a36")
         )
         self.assertTrue(self.pull.update_branch())
+
+    def testDeleteOnMerge(self):
+        self.assertTrue(
+            self.delete_restore_repo.get_branch(self.delete_restore_pull.head.ref)
+        )
+        self.assertFalse(self.delete_restore_pull.is_merged())
+        status = self.delete_restore_pull.merge(deletebranch=True)
+        self.assertTrue(status.merged)
+        self.assertTrue(self.delete_restore_pull.is_merged())
+        with self.assertRaises(github.GithubException) as raisedexp:
+            self.delete_restore_repo.get_branch(self.delete_restore_pull.head.ref)
+        self.assertEqual(raisedexp.exception.status, 404)
+        self.assertEqual(
```

Comment:
Check the raised exception data once, I don't see the need to repeat it everywhere.

Reply:
Understood, would you say we remove line 421 or remove lines 422-429?